mr/src/main/java/org/apache/iceberg/mr/Catalogs.java

```diff
@@ -202,6 +202,16 @@ public static boolean dropTable(Configuration conf, Properties props) {
    */
   public static boolean hiveCatalog(Configuration conf, Properties props) {
     String catalogName = props.getProperty(InputFormatConfig.CATALOG_NAME);
+    return hiveCatalog(conf, catalogName);
+  }
+
+  /**
+   * Returns true if HiveCatalog is used
+   * @param conf a Hadoop conf
+   * @param catalogName name of the catalog
+   * @return true if the Catalog is HiveCatalog
+   */
+  public static boolean hiveCatalog(Configuration conf, String catalogName) {
```

Comment:
is this change still needed then with the new approach?

Reply:
ah it is not needed. Thanks for spotting it!